### PR TITLE
Add more support information of input element

### PR DESCRIPTION
**Pull request by SimonHeimberg**
start adding compatibility information in new format
from deleted old table in https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input$revision/1435221

### CODE REVIEW

**Comment by SimonHeimberg** [comment]
Before adding more data, I would like to get some feedback if I am doing this correctly.
More lines will be added.

**Comment by SimonHeimberg** [comment]
travis is pleased now. Some human feedback to the questions above?

File: html/elements/input.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
+{
+  "html": {
+    "elements": {
+      "input": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/input",
+          "description": "Basic support",
+          "support": {
+            "chrome": {
+              "version_added": "1"
```
Comment (jpmedley) [inline html/elements/input.json]: Please add `chrome_android` and `webview_android`. Use `true` for their values.

Comment (SimonHeimberg) [reply html/elements/input.json]: done in f8350e3

**Comment by wbamberg** [comment]
So sorry to take so long to comment on this. We had a conversation about how to model BCD for input elements a while ago, the main comment is here: https://github.com/mdn/browser-compat-data/pull/1698#issuecomment-390289031.

Essentially, the idea is to treat different input types more or less as if they were different elements, which seems to mesh better with how people treat them.

We would then not document attribute support at the `input` level, we'd document it at the level of the different types. (However we wouldn't document compat for the other attributes unless we need to (meaning, unless the compat story for the other attributes is different from the compat story for the type itself).)

At the moment this gives us compat tables like:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/color#Browser_compatibility
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range#Browser_compatibility
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Browser_compatibility

...which seem quite usable to me.

I'm not against revisiting this choice if we think it's not working out well though.

**Comment by ddbeck** [comment]
Hi @SimonHeimberg, do you plan on responding or updating your PR based on the feedback from wbamberg?

**Comment by ddbeck** [comment]
Hi again, @SimonHeimberg. Thanks for your work on this. I'm assuming that you don't have time to get to this right now. I'm going to close this PR, but please feel free to reopen this in the future, if you get time to do so. Thanks again!

**Pull request closed without merging**